src/components/common/bullets-colin-link.vue

```diff
@@ -0,0 +1,212 @@
+    if (entityType === 'BC' || entityType === 'CC') return entityType
+    if (entityType === 'CR') return 'BEN'
+    if (entityType === 'UL') return 'ULC'
+    return ''
+  }
+
+  // Navigate to the Entity Dashboard.
+  goToEntityDashboard (businessId: string): void {
+    if (businessId) {
+      const dashboardUrl = sessionStorage.getItem('DASHBOARD_URL')
+      Navigate(`${dashboardUrl}${businessId}`)
+    }
+  }
+
+  /**
+   * If user is authenticated, create draft business and redirect to Dashboard.
+   * If user is not authenticated, redirect to login screen then redirect back.
+   */
+  async incorporateNow () {
+    if (this.getIsAuthenticated) {
+      try {
+        // show spinner since this is a network call
+        this.$root.$emit('showSpinner', true)
+        const accountId = +JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id || 0
+        const businessId = await this.createBusiness(accountId)
+        this.goToEntityDashboard(businessId)
+      } catch (error) {
+        this.$root.$emit('showSpinner', false)
+        throw error
+      }
+    } else {
+      const registryHomeUrl = sessionStorage.getItem('REGISTRY_HOME_URL')
+      const nameRequestUrl = `${window.location.origin}`
+      Navigate(`${registryHomeUrl}login?return=${nameRequestUrl}`)
+    }
+  }
+
+  /**
+   * Create a draft business based on selected business type (If applicable).
+   * @param accountId Account ID of logged in user.
+   */
+  async createBusiness (accountId: number): Promise<string> {
+    const legalType = this.entityTypeAlternateShort(this.businessType)
+    const businessRequest = {
+      filing: {
+        header: {
+          name: 'incorporationApplication',
+          accountId: accountId
+        },
+        business: {
+          legalType: legalType
+        },
+        incorporationApplication: {
+          nameRequest: {
+            legalType: legalType
+          }
+        }
+      }
+    } as BusinessRequest
+
+    const createBusinessResponse =
+      await BusinessServices.createBusiness(businessRequest).catch(error => {
+        throw error
+      })
+
+    return createBusinessResponse.data?.filing?.business?.identifier as string
+  }
+
+  // Emit the selected radio button CompanyType enum value.
+  @Emit('radioButtonChange')
+  radioButtonChanged (event: CompanyType): CompanyType {
+    return event
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+// Radio button group spacing
+.v-input--selection-controls {
+  padding-top: 0rem;
+  margin-top: 0rem;
+}
+
+// Remove background color from radio button group
+.v-input__slot, .v-input--selection-controls .v-radio {
+    background-color: white;
+    font-weight: bold;
+}
+
+// Line spacing between bullet points
+.bullet-points {
+  line-height: 2rem;
+}
+
+// Font weight + size of button. This is done to stay consistent with "Check this Name" Button.
+.v-btn {
+  font-weight: bold;
+  min-height: 45px;
```

Comment:
This value is the same value for the button in `search.vue`. I thought it would be a good idea to keep values consistent.